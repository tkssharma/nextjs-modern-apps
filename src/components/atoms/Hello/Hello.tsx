/**
 *
 * Hello
 *
 */
import React from 'react';
import PropTypes from 'prop-types';

const Hello = ({ children }: any) => {

  return (<div>{children}</div>);
};

Hello.propTypes = {
  children: PropTypes.oneOfType([
              PropTypes.arrayOf(PropTypes.node),
              PropTypes.node
            ]).isRequired
}

export default Hello;
