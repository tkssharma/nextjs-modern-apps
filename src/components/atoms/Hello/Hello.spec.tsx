import React from 'react';
import { render } from '@testing-library/react'

import Hello from './Hello';

describe('Hello', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <Hello children=""/>);
    expect(asFragment()).toMatchSnapshot();
  });
});