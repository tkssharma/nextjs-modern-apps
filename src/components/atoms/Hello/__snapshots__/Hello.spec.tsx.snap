// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should render correctly 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
